### PR TITLE
Unapologetic Diona Nerfs: Part 4 of 8274

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Actually reduces nymph health to 33.3. Arcane shitcode set it to 200.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!bugfix

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/mob/living/carbon/alien/diona/diona_nymph.dm [hunk 1]
```diff
@@ -172,7 +173,8 @@
 
 	icon_state = lowertext(species.name)
 	species.handle_post_spawn(src)
-	maxHealth = species.total_health
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/alien/diona/diona_nymph.dm:175]: both of these are unnecessary, no?

File: code/modules/mob/living/carbon/alien/diona/diona_nymph.dm [hunk 2]
```diff
@@ -172,7 +173,7 @@
...
+	health = 33.3
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/alien/diona/diona_nymph.dm:176]: why, I am sure this is already done in mob init

Comment (ParadoxSpace) [reply code/modules/mob/living/carbon/alien/diona/diona_nymph.dm]: done

**Pull request merged**